doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,26 @@
+= Reference points and coordinate systems
+
+With Open Simulation Interface, an object's position coordinates refer to the center of the object's bounding box.
```

Comment:
```suggestion
In Open Simulation Interface, object's position is defined by the coordinates of it's center of bounding box.
```

Reply:
See https://github.com/OpenSimulationInterface/open-simulation-interface/pull/516 for current changes.